UseCases/Sources/HTTPEndToEnd/main.swift

```diff
@@ -0,0 +1,81 @@
+import AsyncHTTPClient
+import Baggage
+import Foundation
+import Instrumentation
+import Logging
+import NIO
+import NIOHTTP1
+
+import NIO
+
+let eventLoopGroup = MultiThreadedEventLoopGroup(numberOfThreads: System.coreCount)
+let threadPool = NIOThreadPool(numberOfThreads: 6)
+threadPool.start()
+
+let orderServiceBootstrap = ServerBootstrap(group: eventLoopGroup)
```

Comment:
Minor:

would be nice to keep the noise out of the "first" lines, so we could:

```
let orderServiceBootstrap = serviceBootstrap(handler:
    OrderServiceHandler(httpClient: httpClient, instrument: FakeTracer())
)
let storageServiceBootstrap = serviceBootstrap(handler:
    StorageServiceHandler()
)
```

with the setup lower in the file (at the end):

```

// MARK: - Setups

func serviceBootstrap(handler: ChannelHandler) -> ServerBootstrap {
    ServerBootstrap(group: eventLoopGroup)
        .serverChannelOption(ChannelOptions.backlog, value: 256)
        .serverChannelOption(ChannelOptions.socketOption(.so_reuseaddr), value: 1)
        .childChannelInitializer { channel in
            channel.pipeline.configureHTTPServerPipeline().flatMap {
                channel.pipeline.addHandler(handler)
            }
        }
        .childChannelOption(ChannelOptions.socketOption(.so_reuseaddr), value: 1)
        .childChannelOption(ChannelOptions.maxMessagesPerRead, value: 1)
        .childChannelOption(ChannelOptions.allowRemoteHalfClosure, value: true)
}


```

Reply:
Good idea 👌